internal/fs/monitoring_fs.go

```diff
@@ -42,6 +42,15 @@ var (
 			"method",
 		},
 	)
+	latencyLookUpInode = prometheus.NewHistogram(
+		prometheus.HistogramOpts{
+			Name: "gcsfuse_fs_look_up_inode_latency",
+			Help: "The latency of executing an LookUpInode request in ms.",
```

Comment:
Let's make it a bit more generic. This way, we can reuse the downstream aggregation templates.

Something like:
latency = prometheus.NewHistogram(
		prometheus.HistogramOpts{
			Name: "gcsfuse_latency",

Add a label for the type. Maybe operation="lookup_inode"?